moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -497,7 +528,7 @@ bool PlanningSceneMonitor::requestPlanningSceneState(const std::string& service_
     throw std::runtime_error("requestPlanningSceneState() to self-provided service: " + service_name);
   }
   // use global namespace for service
-  auto client = pnode_->create_client<moveit_msgs::srv::GetPlanningScene>(service_name);
+  auto client = node_->create_client<moveit_msgs::srv::GetPlanningScene>(service_name);
```

Comment:
Please revert, these and below should be `pnode_`

Reply:
reverted 44c1643